adapter-base/src/main/java/org/eclipse/hono/adapter/MessagingClientSet.java

```diff
@@ -34,12 +34,12 @@
 /**
  * The set of messaging clients present in a protocol adapter.
  * <p>
- * It contains one client for each connected messaging system and allows to get the messaging to be used for a tenant
- * based on the tenants configuration.
+ * It contains one client for each connected messaging system type and allows to get the messaging to be used for a
+ * tenant based on the tenants configuration.
  */
-public class MessagingClients implements Lifecycle {
+public final class MessagingClientSet implements Lifecycle {
```

Comment:
FMPOV you renamed the wrong class. IIRC I had proposed to rename `MessageClient` to `MessageClientSet` because it holds a set of clients. `MessageClients` can remain as-is FMPOV ...

Reply:
Oh yes, I see now, your proposal was to rename the other class. It made perfect sense to me :laughing: 
I'll change it.